tests/mappings/mapping_q_eulerian_02.cc

```diff
@@ -105,8 +105,7 @@ void test ()
   // now the actual test
   DataOut<dim> data_out;
   data_out.attach_dof_handler(dof_handler);
-  std::vector<std::string> names (dim, "displacement");
-  data_out.add_data_vector(displacements,names);
+  data_out.add_data_vector(displacements, "displacement");
```

Comment:
Hm, are all of these places really wrong? The underlying DoFHandler has a FE that consists of dim copies of the same element. Or is this wrong because of the lack of a specification that the components logically form a vector via a `DataComponentInterpretation` argument?